tslearn/testing_utils.py

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
I think this should be a test file. That means, the file name should be: `tslearn/tests/test_estimators.py` and it should not be seen as a `tslearn` module.

Also `check_all_estimators` should be `test_all_estimators` and should raise an assertion upon failure.

Reply:
I am unsure what `check_estimator` from `sklearn` does, maybe it already raises exceptions upon failure